.github/workflows/_tox-unit-test.yaml

```diff
@@ -2,30 +2,37 @@ name: Unit test python code
 on:
   workflow_call:
     inputs:
-      python-versions:
-        default: "3.9,3.10,3.11,3.12"
+      python-version:
```

Comment:
is inputs.python-version actually used anywhere?

Reply:
It is. The composite action takes it as input. This way, the caller can specify the version of python and poetry without influencing the other repositories